src/api/routes/authRouter.js

```diff
@@ -45,6 +46,7 @@ authRouter.get('/login/twitter', passport.authenticate('twitter', { scope: ['pro
 authRouter.get('/login/twitter/redirect', passport.authenticate('twitter', { session: false }), twitter, socialLogin.twitterLogin);
 
 authRouter.post('/signup', validateBody('signup'), validateGender, usernameExists, emailExists, register);
+authRouter.post('/login', login);
```

Comment:
You can make use of the middleware here like this: `validateBody('signin')`. Where `'signin'` is a schema that will be added to `helpers/validationSchemas.js`. I explained this on the slack channel but I am willing to help if you need it.